Language changes

ryanctan opened the pull request:
Hopefully all of the changes are cosmetic, please review to make sure nothing is broken

ryanctan committed:
I put hi

ryanctan committed:
Wording changes for GTE explorer page

Minor language changes for some terms.

ryanctan committed:
Revert "I put hi"

This reverts commit 675592770b445d11e6eb1dfd876a10b110d3a7f1.

ryanctan committed:
More language changes

ryanctan committed:
employment or identifier polishing

ryanctan commented:
Changes in the Explorer and Search finished. Still working on new entries and tsv output. 

@ceserani 
1. Within the query itself, variable names like fullName and mentionedNames are still visible (e.g. after entering a name). Is this something I should change as well? 
2. And I believe the export results cannot be reordered with Travelers_Life_Events before Travelers_Itineraries because it is just sorted alphabetically by file type.

ryanctan committed:
All tsv language changes

ryanctan commented:
@epicfaace all language changes are submitted and ready to be reviewed

ryanctan committed:
tsv language changes, bug-free

epicfaace commented on public/register/register.pug:
```diff
@@ -8,9 +8,9 @@
           input.form-control(type="text", name="username", ng-model="user.username", placeholder="User name", required)
       .form-group
-        label.col-sm-3.control-label Full name
+        label.col-sm-3.control-label Traveler names
```
Why "Traveler names" but not "Traveler name"?

ryanctan replied:
Just submitted a new commit -- I actually think this "Full name" actually refers to a user registering for the website, so it's been restored

ryanctan committed:
register full name restored

I believe this program is meant to display for a user signing up on the website, so full name was restored

ryanctan committed:
fullName restored in admin

epicfaace commented on query.js:
```diff
@@ -627,33 +627,33 @@ function parseExport(res) {
-            value: a.title,
+            lifeEvents: 'occupation',
+            eventsDetail1: a.group,
+            eventsDetail2: a.title,
```
@ryanctan I noticed you changed the names for the columns to export here; could you explain a bit why you did so? I'm assuming it's for readability's sake, but just want to make sure everyone's on board with this before we merge (as this could make people who work with the data currently have to change their workflows a bit since the column names have changed).

ryanctan replied:
Changes were made based on #73

epicfaace replied:
@ryanctan I believe the intention (at least the last time @ceserani and I discussed this) was probably for column names in the exported tsv files to have spaces in them. i.e., "Life events" instead of "lifeEvents", "Events Detail 1" instead of "eventsDetail1", etc. That way, the tsv column names are more human-readable. We can probably make another issue for this though.

epicfaace reviewed (commented):
Everything looks good, can you just clarify on that one point about exports?
